src/involvement/views/position_edit_view.py

```diff
@@ -1,14 +1,13 @@
 from wagtail.contrib.modeladmin.views import EditView
 from involvement.models import Role
-from involvement.rules import is_admin
 
 
 class PositionEditView(EditView):
     def get_form(self, form_class=None):
         form = super(PositionEditView, self).get_form(form_class=form_class)
         queryset = form.fields['role'].queryset
-        if not is_admin(self.request.user):
-            queryset = Role.edit_permission_of(self.request.user)
+        if not self.request.user.is_superuser:
```

Comment:
Why use the direct Django permissions instead of the permission that have just been implemented? This seems a bit of a risk if the permissions should ever change. Should `is_super` be used?

Otherwise maybe that function should be removed and the direct system should always be used.

Reply:
I think consistency in this case would be very important. if you want to use `is_super` I suggest you have to consider every occurrence of `is_superuser`.

Considering this (on second thought) I think I actually prefer using Django's builtin function. I think it leaves things more maintainable.